server/services/licenceService.ts

```diff
@@ -573,15 +573,6 @@ export default class LicenceService {
     return this.licenceApiClient.deleteOmuEmailAddress(prisonId, user)
   }
 
-  async getParentLicenceOrSelf(licenceId: string, user: User): Promise<Licence> {
```

Comment:
Can we keep this here but delegate down to the licenceApiClient? 

Controllers shouldn't be directly talking to the client, can swap those back to use the licence service

Reply:
I believe this had to be done because otherwise it caused a circular dependency between conditionService and licenceService, but hopefully Nish can confirm (I may be making that up 😅)

Reply:
Only solution is to duplicate the method in both licenceService and licenceApiClient. Can I do that?

Reply:
I am delegating the getParentLicenceOrSelf method through licenceApiClient So the controllers can communicate through service.